### ISSUE TITLE
Crafting Menu + Msg

### ISSUE THREAD

**Issue by xushTW** [CytraL/MineTee#5]
Maybe it would be nicer to craft items in a GUI instead of via the chat.

Also a new NetMsg for this would probably be a good idea.

**Comment by CytraL** [comment]
There exists one problem... NetMsg are limited... if one client like have compatibility with the mod and for example with ddracenetwork too... it can't... because can't listen all NetMsg.

But perhaps is best not think in other compatibility...

**Comment by xushTW** [comment]
Ofc the compatibility can be broken. The vanilla client will never be supported anyways obviously.

**Comment by CytraL** [comment]
yeah, is not for vanilla client... more ppl make you own client with parts of all "active" clients... but... ok... break with all! i think the same... really nobody go to make your client compatible with this mod xDD
